.circleci/config.yml

```diff
@@ -69,6 +70,57 @@ commands:
           name: print installed packages
           command: pip freeze --all
 
+  dependencies-windows:
+    # You must still manually update the Docker image if any
+    # binary (non-Python) dependencies change.
+    description: "Check out and update Python dependencies."
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - v6-win-dependencies-{{ checksum "setup.py" }}
+
+      - run:
+          name: install python
+          command: |
+            $ErrorActionPreference = "Stop"  # exit immediately on any error
```

Comment:
Is this needed when there's only one command?

Reply:
I don't think it is needed, but I added it anyway since it doesn't do any harm. Should I remove it?

Reply:
I lean towards removing it as we do not include it in any of the following one-liner commands.

There is a case for keeping it as defensive coding in case we add multiple commands later, but in that case I think we should do this elsewhere as well.

We could perhaps make Stop the default and eliminate the need for this in every script block? https://github.com/hashicorp/packer/issues/6449#issuecomment-402586809 has a method of doing this, though it seems like it needs admin permissions (perhaps there is a local-user version of this too though).

Reply:
I checked the link's command through CircleCI's ssh option, and it seems to be working. I'm guessing CircleCI provides shell with Admin privileges. 
So I have added the command from the link in the config.